From c1e8ccae3cdf1803558559247522d368f55f5f45 Mon Sep 17 00:00:00 2001
From: adryd <me@adryd.com>
Date: Sat, 26 Feb 2022 19:41:22 -0500
Subject: [PATCH] 42

---
 metadata.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index dfca6ed..285a26c 100644
--- a/metadata.json
+++ b/metadata.json
@@ -8,6 +8,7 @@
     "3.36",
     "3.38",
     "40",
-    "41"
+    "41",
+    "42"
   ]
 }
--
2.35.1

